templates/pages/index.html

```diff
@@ -5,9 +5,13 @@
 {% block content %}
 	<!-- Main jumbotron for a primary message -->
 	<div class="jumbotron">
-		<h1>TrackCat</h1>
+        <div class="h1"> 
```

Comment:
could you style the h1 tag? give it a class (or id), and target that tag specifically